pkgs/development/libraries/cosmopolitan/default.nix

```
@@ -0,0 +1,68 @@
+{ lib, gcc9Stdenv, fetchFromGitHub, runCommand, cosmopolitan }:
+
+gcc9Stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+  src = fetchFromGitHub {
+    owner = "jart";
+    repo = "cosmopolitan";
+    rev = "94afa982c35d4322999b5591f6d41d5cfc3f19a5";
+    sha256 = "sha256-LaXkFMoArg5LUw4AdyhOqxzSJLdQCwHVBK2+lXVmxuI=";
+  };
+
+  postPatch = ''
+    patchShebangs build/
+    rm -r third_party/gcc
+  '';
+
+  dontConfigure = true;
+  dontFixup = true;
+  enableParallelBuilding = true;
+
+  preBuild = ''
+    makeFlagsArray=(SHELL=/bin/sh AS=as CC=gcc GCC=gcc CXX=g++ LD=ld OBJCOPY=objcopy "MKDIR=mkdir -p")
```

```suggestion
    makeFlagsArray=(SHELL=/bin/sh AS=${stdenv.cc.targetPrefix}as CC=${stdenv.cc.targetPrefix}gcc GCC=${stdenv.cc.targetPrefix}gcc CXX=${stdenv.cc.targetPrefix}g++ LD=${stdenv.cc.targetPrefix}ld OBJCOPY=${stdenv.cc.targetPrefix}objcopy "MKDIR=mkdir -p")
```

For cross-compilation.